package.json

```diff
@@ -11,21 +11,20 @@
     "dependencies": {
         "artedi": "1.1.0",
         "assert-plus": "1.0.0",
-        "bignum": "0.6.0",
+        "bignum": "0.13.0",
         "bunyan": "0.22.1",
         "clone": "0.1.9",
-        "dtrace-provider": "0.2.8",
-        "fash": "2.5.0",
+        "dtrace-provider": "0.8.8",
+        "fash": "git+https://github.com/joyent/node-fash.git#MANTA-4720",
```

Comment:
I think we should try to sort this out (get npm publish rights, and push and publish a fix) rather than locking to a temporary branch.

Reply:
sounds good. as per the chat, this is in the process currently with jclulow